RptToXml/Program.cs

```diff
@@ -62,19 +66,21 @@ static void Main(string[] args)
 			{
 				try
 				{
-					Trace.WriteLine("Dumping " + rptPath);
+					if(!stdOut)
+						Trace.WriteLine("Dumping " + rptPath);
+					
 
-					using (var writer = new RptDefinitionWriter(rptPath))
+					using (var writer = new RptDefinitionWriter(rptPath, stdOut))
 					{
-						string xmlPath = args.Length > 1 + ignoreErrFlag ?
+						string xmlPath = args.Length > 1 && !ignoreErrFlag ?

```

Comment:
Seems we should look at stdOut flag here and then either call WriteToXml(xmlPath) or WriteToXml() rather than the RptDefinitionWriter being aware of stdOut flag.

Reply:
@ajryan I did it through the def writer so that the print statements could be suppressed, particularly this one: https://github.com/ajryan/RptToXml/blob/f80f937726690f237b8061c5fe048cfc960cc014/RptToXml/RptDefinitionWriter.cs#L42

This could be moved into the WriteToXml(path), but then is not really useful.

What do you think?